fe1-web/src/features/popcha/screens/PopchaScanner.tsx

```diff
@@ -148,42 +164,88 @@ const PopchaScanner = () => {
       });
   };
 
-  return (
+  return showScanner ? (
     <>
       <QrCodeScanner
-        showCamera={showScanner}
+        showCamera
         handleScan={(data: string | null) =>
           data &&
           sendAuthRequest(data).then((success) => {
             if (success) {
-              setTextScanned(data);
               setShowScanner(false);
+              showSuccessMessage(STRINGS.popcha_success_authentication);
             }
           })
         }>
+        <View style={styles.container}>
+          <View />
```

Comment:
Can't you use flex `space-around` to achieve what you want ?

Reply:
not sure how it would change to use `space-around` in this case?

Reply:
If the top third needs to be empty, then it won't work. But using some % margins with flex-grow might work, I guess. But it might be even less readable than the current state.